Index: lingua_idiom/src/components/header.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import React, { Component } from \"react\";\r\n//Импорт картинок\r\nimport logoHeader from \"../img/logo_header_logout.png\";\r\n\r\nclass Header extends React.Component {\r\n  render() {\r\n    return (\r\n      <div className=\"header\">\r\n        <div className=\"headerInside\">\r\n          <img src={logoHeader} />\r\n        </div>\r\n        <div className=\"buttons\">{this.props.buttons}</div>\r\n      </div>\r\n    );\r\n  }\r\n}\r\nexport default Header;\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/lingua_idiom/src/components/header.js b/lingua_idiom/src/components/header.js
--- a/lingua_idiom/src/components/header.js	
+++ b/lingua_idiom/src/components/header.js	
@@ -6,7 +6,7 @@
   render() {
     return (
       <div className="header">
-        <div className="headerInside">
+        <div className="headerChildren">
           <img src={logoHeader} />
         </div>
         <div className="buttons">{this.props.buttons}</div>
Index: lingua_idiom/src/components/PageBodu.js
===================================================================
diff --git a/lingua_idiom/src/components/PageBodu.js b/lingua_idiom/src/components/PageBodu.js
deleted file mode 100644
--- a/lingua_idiom/src/components/PageBodu.js	
+++ /dev/null	
@@ -1,37 +0,0 @@
-import React from "react";
-import { Layout } from "antd";
-import page_body from "../img/page_body_right.png";
-import Ellipse from "../img/Ellipse.png";
-import Vector from "../img/Vector.png";
-const { Content } = Layout;
-
-// npm i antd что бы работало с компонентами
-class PageBodu extends React.Component {
-  render() {
-    return (
-      <div className="PageBobi">
-        <div className="Contentleft">
-          <h1 className="page_body_p1">Откройте мир фразеологизмов!</h1>
-          <h4 className="page_body_p2">
-            Мы предлагаем платформу для тех, кто занимаеться изучением
-            французского или корейского языков и столкнулся с проблемой перевода
-            фразеологизмов и идиом
-          </h4>
-          <div className="page_button ">
-            <p className="page_button_text">Начать переводить</p>
-            <div className="page_button_circle">
-                 <img className="" src={Ellipse} />               
-            </div>              
-          </div>
-          
-        </div>
-        <div className="Contentright">
-          <div>
-            <img src={page_body} />
-          </div>
-        </div>
-      </div>
-    );
-  }
-}
-export default PageBodu;
